Library/Homebrew/shims/scm/git

```diff
@@ -9,6 +9,12 @@ quiet_safe_cd() {
   cd "$1" >/dev/null || { echo "Error: failed to cd to $1" >&2; exit 1; }
 }
 
+abspath() {
+  local dir="$(quiet_safe_cd "${1%/*}/" && pwd -P)"
```

Comment:
@UniqMartin As usual, I simultaneously thank you for the review and hate you for pointing out an edge case 😆. PR updated!